docs/user_guide/basic_concepts.rst

```diff
@@ -29,7 +29,7 @@ This definition is close, but not equivalent, to the definition used by ``cirq``
 the main difference is that ``cirq`` defines a moment as a sub-circuit of depth 1: no
 instruction in the moment overlaps with another instruction in the same moment.
 In the definition given above, there is no notion of depth, and even though ``TICK``
-instruction are generally used in ``stim`` such as the moment definition above
+instruction are generally used in ``stim`` to denote the passage of time such as the moment definition above
 is equivalent to the definition used by ``cirq``, this is not required.
```

Comment:
@inmzhang Do you happen to understand what this sentence is trying to say? I am confused about what `this is not required` is referring to.